openpype/hosts/houdini/plugins/publish/validate_frame_range.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from openpype.hosts.houdini.api.action import SelectInvalidAction
+
+import hou
+
+
+class ValidateFrameRange(pyblish.api.InstancePlugin):
+    """Validate Frame Range.
+
+    Due to the usage of start and end handles,
+    then Frame Range must be >= (start handle + end handle)
+    which results that frameEnd be smaller than frameStart
+    """
+
+    order = pyblish.api.ValidatorOrder - 0.1
+    hosts = ["houdini"]
+    label = "Validate Frame Range"
+    actions = [SelectInvalidAction]
+
+    def process(self, instance):
+
+        invalid = self.get_invalid(instance)
+        if invalid:
+            nodes = [n.path() for n in invalid]
+            raise PublishValidationError(
+                "Invalid Frame Range on: {0}".format(nodes),
+                title="Invalid Frame Range"
+            )
```

Comment:
On the `PublishValidationError` let's make sure we add a `description` that also explains what the Validator does and explains what one could do about it:

> ### Invalid frame range
> 
> The frame range for the instance is invalid because the start frame is higher than the end frame.
>
> This is likely due to asset handles being applied to your instance or may be because the ROP node's
> start frame is set higher than the end frame.
>
> If your ROP frame range is correct and you do not want to apply asset handles make sure 
> to disable *Use asset handles* on the publish instance.

Reply:
This is awesome! Thanks! 
![image](https://github.com/ynput/OpenPype/assets/20871534/afd9580b-042b-4900-a0de-51243314eb08)